tech/languages/ocaml/ocaml-installation.md

```diff
@@ -0,0 +1,49 @@
+---
+title: OCaml
+subsection: ocaml
+section: tech-languages
+order: 1
+version: 1.0
+description: An open source multi-paradigm programming language and toolchain. 
+---
+
+# What is OCaml?
+
+OCaml is an industrial-strength programming language supporting functional, imperative and object-oriented styles.
+
+## OCaml installation
+
+To install the OCaml tools, type on a terminal:
+
+```console
+$ sudo dnf install ocaml ocaml-findlib 
+```
+
+The REPL (Read-Evaluate-Print-Loop) tool (`ocaml`), the library manager ( `ocamlfind`), the bytecode compiler (`ocamlc`), and the native-code compiler binaries (`ocamlopt`) will become available on the system. 
+
+# OPAM
+
+OPAM (**O**Caml **Pa**ckage **M**anager)  is a source-based package manager for OCaml. It supports multiple simultaneous compiler installations, flexible package constraints, and a Git-friendly development workflow.
+
+To install OPAM in your Fedora just:
+```console
+$ sudo dnf install opam
+```
+
+To use OPAM to install and manage your OCaml instalation just:
+```console
+# environment setup
+$ opam init
```

Comment:
Running this command gives me some options, could you explain in more detail here what the user can expect and what options are there? (Or provide a link where user can read more specifically about `opam init`)